pom.xml

```diff
@@ -19,7 +19,7 @@ LICENSE file.
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>site.ycsb</groupId>
+  <groupId>org.gridgain.ycsb</groupId>
   <artifactId>root</artifactId>
   <version>0.18.0-SNAPSHOT</version>
```

Comment:
Do we need to set an internal version like `2023.8`?

Reply:
+1 for using custom version number